UI/src/app/home/home.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { JwtHelperService } from '@auth0/angular-jwt';
+
+@Component({
+  selector: 'app-home',
+  templateUrl: './home.component.html',
+  styleUrls: ['./home.component.sass']
+})
+export class HomeComponent implements OnInit {
+
+  constructor(private jwtHelper: JwtHelperService, private router: Router) { }
+
+  ngOnInit(): void {
+  }
+
+  isUserAuthenticated() {
```

Comment:
you should use your own AuthenticationService behind operations like these: `isUserAuthenticated` and `logOut`